src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -208,18 +304,23 @@ class TypeAheadVariableDropdown extends PureComponent<Props, MyState> {
             onCollapse={onCollapse}
             theme={DropdownMenuTheme.Amethyst}
           >
-            {shownValues.map(val => {
+            {shownValues.map(function(value, index) {
+              let classN = 'variable-dropdown--item'
+              // highlight when arrowing; like a hover
+              if (index === selectIndex) {
+                classN += ' active'
+              }
```

Comment:
we use `classnames` as a dependency, it's a great tool for this kind of stuff: https://jedwatson.github.io/classnames/

Reply:
cool! using it